Neuro/Movement/MovementHandler.cs

```diff
@@ -86,6 +91,57 @@ public void GetForcedMoveDirection(ref Vector2 direction)
         renderer.SetPosition(1, PlayerControl.LocalPlayer.GetTruePosition() + direction);
     }
 
+    public void MoveDeadPlayer() {
+        Console closestConsole = null;
+        float closestDistance = 999f;
+        
+        foreach (NormalPlayerTask task in PlayerControl.LocalPlayer.myTasks.ToArray().OfIl2CppType<NormalPlayerTask>().Where(t => !t.IsComplete))
+        {
+
+            foreach (Console console in task.FindConsoles())
+            {
+                if (!console) continue;
+                var distance = Vector2.Distance(console.transform.position, PlayerControl.LocalPlayer.GetTruePosition());
+                if (distance < closestDistance)
+                {
+                    closestDistance = distance;
+                    closestConsole = console;
+                }
+            }
+        }
+        if (closestConsole) {
+            moveToPosition(closestConsole.transform.position);
+            return;
+        }
+
+        // follow random player
+        if (followPlayer is null || followPlayer.Data.IsDead)
+        {
+            List<PlayerControl> alivePlayers = new();
+            foreach (PlayerControl player in PlayerControl.AllPlayerControls) 
+            {
+                if (!player.Data.IsDead) 
+                {
+                    alivePlayers.Add(player);
+
+                } 
+            }
+            System.Random random = new();
+            int i = random.Next(alivePlayers.Count);
+            followPlayer = alivePlayers[i];
+            Info("Now following " + followPlayer.name);
+        }
+        moveToPosition(followPlayer.GetTruePosition());
```

Comment:
Instead of doing this, use the GhostRole's haunt ability to haunt an impostor

Reply:
CrewmateGhosts now automatically haunt impostors. How should ImpostorGhosts and GuardianAngel be handled? You can't sabotage while haunting and the GuardianAngel doesn't even have the ability to haunt.